### DISCUSSION TITLE
ESP32-s2 and ESP32-c3 support?

### DISCUSSION THREAD

**Discussion by ajvdw** [emsesp/EMS-ESP32#656] [Ideas]
Any thoughts on support for the S2 and the C3 in the source tree as well as separate releases? I think these devices are an easier upgrade to the BBQKees EMS gateway than the ESP32 because of the form factor.

**Comment by proddy** [comment]
See https://github.com/emsesp/EMS-ESP32/issues/605

The S2/C3 have different cores. I haven't tried building against that yet and unsure whether the Arduino libraries support it. You can try the port yourself ?

**Reply by ajvdw** [reply]
See 605
